yotiprofile.go

```diff
@@ -2,104 +2,77 @@ package yoti
 
 import (
 	"github.com/getyoti/yoti-go-sdk/v2/attribute"
-	"github.com/getyoti/yoti-go-sdk/v2/yotiprotoattr"
 )
 
+// Attribute names
 const (
-	attrConstSelfie                  = "selfie"
-	attrConstGivenNames              = "given_names"
-	attrConstFamilyName              = "family_name"
-	attrConstFullName                = "full_name"
-	attrConstMobileNumber            = "phone_number"
-	attrConstEmailAddress            = "email_address"
-	attrConstDateOfBirth             = "date_of_birth"
-	attrConstAddress                 = "postal_address"
-	attrConstStructuredPostalAddress = "structured_postal_address"
-	attrConstGender                  = "gender"
-	attrConstNationality             = "nationality"
-	attrConstDocumentImages          = "document_images"
+	AttrConstSelfie                    = "selfie"
+	AttrConstGivenNames                = "given_names"
+	AttrConstFamilyName                = "family_name"
+	AttrConstFullName                  = "full_name"
+	AttrConstMobileNumber              = "phone_number"
+	AttrConstEmailAddress              = "email_address"
+	AttrConstDateOfBirth               = "date_of_birth"
+	AttrConstAddress                   = "postal_address"
+	AttrConstStructuredPostalAddress   = "structured_postal_address"
+	AttrConstGender                    = "gender"
+	AttrConstNationality               = "nationality"
+	AttrConstDocumentImages            = "document_images"
+	AttrConstDocumentDetails           = "document_details"
+	AttrConstApplicationName           = "application_name"
```

Comment:
Would it make more sense for the ApplicationProfile constants to live in `application_profile.go` rather than here?